process/release_process.md

```diff
@@ -111,6 +111,31 @@ It leads us to need a separate 'xt-core' repository that will be the entry point
     - Merging the 'development' branch into the 'main' branch
     - Adding the commit that locks all the dependencies
     - Assigning the release tag to the HEAD commit of the 'main' branch
+- **Approach #3**:
+   - Development is done in the 'development' branch. In this branch, all dependencies can stick to the HEAD. In this stage we separate repositories that are including external dependencies and requires actual commit to stabilize external revision. They could be:
```

Comment:
"development" branch is "master" branch. Is this correct?

Reply:
See phrase:
>   - Reducing number of branches which requires test to one. Because testing with moulin build requires time and effort

Reply:
In our case we have rpi5_dev or rz_g3s_dev or rz_a2m dev and so on. So I can't answer your question in the way you ask. From semantic standpoint - it is develop branch. But we have product specific branches for zephyr-dom0-xt for example. That's why I propose separate approach because I believe there is no generic approach